README.md

```diff
@@ -101,19 +124,21 @@ Der Konfigurationsblock in der `package.json` sieht wie folgt aus:
   }
 ```
 
-##### 3.2.2.2 Aurelia CLI Support
+#### 3.2.1 Verwendung von Komponenten eines Plugins
 
-Beim bauen des Plugins wird ein AMD Modul exportiert, das die Aurelia CLI verwenden kann.
+Stellen wir uns vor eine Aurelia Applikation möchte das Custom Element `my-example` aus dem `example` Feature dieses Plugin Skeletons verwenden.
 
-Nach dem Import des Plugins wird in der `aurelia.json` der Aurelia Applikation ein Eintrag für das Vendor Bundle erzeugt:
+Die zugehörige View sieht dann wie folgt aus:
+
+```html
+<template>
+  <require from="aurelia_plugin_skeleton/example/elements/my-example"></require>
+
+  <my-example-tag></my-example-tag>

```

Comment:
wäre hier günstiger das tag nach der aurelia namenskonvention zu benennen - in diesem fall `<my-example></my-example>`